src/yamlfix/entrypoints/cli.py

```diff
@@ -20,17 +21,22 @@ def _format_file_list(files: Tuple[TextIOWrapper]) -> str:
 @click.command()
 @click.version_option(version="", message=version.version_info())
 @click.option("--verbose", is_flag=True, help="Enable verbose logging.")
+@click.option("--check", is_flag=True, help="Check if file(s) needs fixing.")
```

Comment:
I'm missing the idea that with `--check` no file is going to be changed in the `help` message

Reply:
Good point! Fixed